src/test/ui/rfc-2632-const-trait-impl/inherent-impl.rs

```diff
@@ -1,13 +1,16 @@
 #![feature(const_trait_impl)]
 #![allow(bare_trait_objects)]
+#![feature(effects)]
 
 struct S;
 trait T {}
 
 impl const S {}
 //~^ ERROR inherent impls cannot be `const`
+//~| ERROR `host` is not constrained
```

Comment:
adding an effect param for inherent impls seems bad ui wise ^^

Reply:
yea I hate everything about it ^^ I'll remove it and just require everyone to repeat trait bounds on the functions for now